quantum/split_common/matrix.c

```diff
@@ -286,6 +286,16 @@ int serial_transaction(void) {
     for (int i = 0; i < ROWS_PER_HAND; ++i) {
         matrix[slaveOffset+i] = serial_slave_buffer[i];
     }
+    
+    #ifdef RGBLIGHT_ENABLE
+        // Code to send RGB over serial goes here (not implemented yet)
```

Comment:
Well, for serial, probably using the second data line for the RGB strips, so probably isn't a big deal

Reply:
True, but put the block there so if in the future it is implemented people know where to put it =P

Reply:
If you'd rather that removed also np, just let me know.

Reply:
No, it should be fine, I think. :)